source/reference/schema.rst

```diff
@@ -21,35 +21,36 @@ keywords.
 $schema
 -------
 
-The ``$schema`` keyword is used to declare which version of the JSON
-Schema specification the schema was written for. The value of the
-``$schema`` keyword is also the identifier for a schema that can be
-used to verify that the schema is valid according to the specification
-version ``$schema`` identifies. A schema that describes another schema
-is called a "meta-schema".
+The ``$schema`` keyword is used to declare which dialect of JSON
+Schema the schema was written for. The value of the ``$schema``
+keyword is also the identifier for a schema that can be used to verify
+that the schema is valid according to the dialect ``$schema``
+identifies. A schema that describes another schema is called a
+"meta-schema".
 
 ``$schema`` applies to the entire document and must be at the root
-level. It does not apply to externally referenced (``$ref``)
-documents. Those schemas need to declare their own ``$schema``.
+level. It does not apply to externally referenced (``$ref``,
+``$recursiveRef``) documents. Those schemas need to declare their own
```

Comment:
```suggestion
``$recursiveRef``, ``$dynamicRef``) documents. Those schemas need to declare their own
```

Reply:
Thanks. I'm sure I would have missed that one. I think it should just be `$dynamicRef` because `$recursiveRef` doesn't exist in 2020-12.

Reply:
fair enough :) I'm happy to see the death of $recursiveRef.